code/__HELPERS/game.dm

```diff
@@ -152,11 +152,11 @@
 /proc/recursive_hear_check(O)
```

Comment:
Oh yea, also this is an unused proc, if you wanna remove.

Reply:
morgue uses it and me lazy